### DISCUSSION TITLE
how do I select deploy region in Rowy?

### DISCUSSION THREAD

**Discussion by simon-muyezhang** [AntlerVC/firetable#883] [Support / Q&A] [answered]
It always uses us-central1.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @simon-zhangmuye, Rowy doesn't currently have a UI to select a deploy region for the GCP functions. You can change these either by accessing the `_rowy_` collection in your Firebase Firestore or using the GCP console.
<img width="718" alt="image" src="https://user-images.githubusercontent.com/53828745/196995114-e92134de-0e81-438c-bc4d-47631a5795e7.png">

**Comment by simon-muyezhang** [comment]
thanks a lot.
